storagenode/peer.go

```diff
@@ -265,6 +271,29 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config Config, ver
 		)
 	}
 
+	{ // setup vouchers
+		if config.Vouchers.Interval <= 0 {
+			return nil, errs.New("voucher service interval (%d) must be > 0", config.Vouchers.Interval)
+		}
+		intervalHours := config.Vouchers.Interval * 24
+		intervalDuration, err := time.ParseDuration(fmt.Sprintf("%dh", intervalHours))
+		if err != nil {
+			return nil, err
+		}
+
+		if config.Vouchers.ExpirationBuffer <= 0 {
+			return nil, errs.New("voucher service expiration buffer (%d) must be > 0", config.Vouchers.ExpirationBuffer)
+		}
+		bufferHours := config.Vouchers.ExpirationBuffer * 24
+		bufferDuration, err := time.ParseDuration(fmt.Sprintf("%dh", bufferHours))
```

Comment:
you don't have to do any of this if you just make the config type be a time.Duration off the bat. our config system knows how to handle time.Durations